easybuild/easyblocks/c/cuda.py

```diff
@@ -92,14 +92,23 @@ def install_step(self):
         elif LooseVersion(self.version) > LooseVersion("5") and LooseVersion(self.version) < LooseVersion("10.1"):
             install_interpreter = "perl"
             install_script = "cuda-installer.pl"
-            # note: also including samples (via "-samplespath=%(installdir)s -samples") would require libglut
+            # note: samples are installed by default
             self.cfg.update('installopts', "-verbose -silent -toolkitpath=%s -toolkit" % self.installdir)
         else:
             install_interpreter = ""
             install_script = "./cuda-installer"
-            # note: also including samples (via "-samplespath=%(installdir)s -samples") would require libglut
-            self.cfg.update('installopts', "--silent --toolkit --toolkitpath=%s --defaultroot=%s" % (
-                            self.installdir, self.installdir))
+            # samples are installed in two places with identical copies:
+            # self.installdir/samples and $HOME/NVIDIA_CUDA-11.2_Samples
```

Comment:
Can we avoid the `$HOME` issue by redefining `$HOME` to `self.builddir` or something like that?

Reply:
If we do that, make sure that the scope is as short-lived as possible as it has the potential to mess with things like out GitHub integration.

Reply:
$HOME should rather be redirected to a tmpdir, not the builddir.

Reply:
The comment could be a little clearer: "second location" below refers to "$HOME/NVIDIA_CUDA-11.2_Samples". I'll do that then.

Reply:
Even better might be "changing the second location (the one under $HOME) to a scratch location using --samples --samplespath=self.builddir" because the comment doesn't make it clear that this is what's being done....